src/herbpy/herbrobot.py

```diff
@@ -30,6 +34,32 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
 
         Robot.__init__(self, robot_name='herb')
 
+
+        # Controller setup
+        # TODO load controller names from same config as ros_control?
+        self.controller_manager = None  # TODO verify never used in sim
+        self.controllers_always_on = []
+        controllers_manip = []
+
+        # TODO how to deal with sim on controller manager?
```

Comment:
The controller manager does not exist in simulation, so there is (hopefully) nothing to deal with. If that is the case, delete this `TODO` message.